docs/data/tree-view/simple-tree-view/customization/customization.md

```diff
@@ -11,11 +11,51 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/treeview/
 
 <p class="description">Customize the rendering of your Tree View.</p>
 
-## Custom icons, border and animation
+## Basics of customization
+
+### Custom icons
+
+Use the `defaultCollapseIcon`, `defaultExpandIcon`, and `defaultEndIcon` props to customize the Tree View icons. You can use either icon libraries, such as Material Icons, or create your own.
+
+{{"demo": "CustomIcons.js", "defaultCodeOpen": false}}
+
+### Custom toggle animations
+
+Use the `TransitionComponent` prop on the `TreeItem` to pass a component that handles your animation.
+
+{{"demo": "CustomAnimation.js", "defaultCodeOpen": false}}
+
+### Styling
+
+You can use the `treeItemClasses` to target the internal elements of the TreeView.
+
+{{"demo": "CustomStyling.js"}}
+
+### Setting custom content
+
+You can use the `ContentComponent` prop to replace the `TreeItem` with a custom component
+
+{{"demo": "CustomContentTreeView.js"}}
+
+## Common examples
+
+### Connection border
+
+Target the `treeItemClasses.group` class to add connection borders between the Tree View items.
+
+{{"demo": "BorderedTreeView.js", "defaultCodeOpen": false}}
+
+### File explorer
+
+You can customize the Tree View to accomodate more complex use cases:
 
 {{"demo": "CustomizedTreeView.js"}}
 
-## ContentComponent prop
+### Gmail clone
+
+{{"demo": "GmailTreeView.js"}}
+
+### ContentComponent prop
```

Comment:
Feeling like this being part of the Common examples is not fitting... also given we have already introduced this prop on the last demo of the Basics section. How about adding just one of the demo below, right after the "Adding custom content" demo, and dedicated to the `useTreeItem` hook?

Reply:
Hmm, I think the demo in the basics section already uses the `useTreeItem` hook. How about rephrasing the description there, and rebranding this one like Flavien suggested? (Did it in the latest commit)

Reply:
I think that works! Throwing in an additional round of comments super quick :)